app/src/main/java/fr/free/nrw/commons/upload/worker/UploadWorker.kt

```diff
@@ -46,13 +48,15 @@ import timber.log.Timber
 import java.util.*
 import java.util.regex.Pattern
 import javax.inject.Inject
-import kotlin.collections.ArrayList
+
 
 class UploadWorker(var appContext: Context, workerParams: WorkerParameters) :
     CoroutineWorker(appContext, workerParams) {
 
     private var notificationManager: NotificationManagerCompat? = null
 
+    @Inject
+    lateinit var dbOpenHelper: DBOpenHelper
```

Comment:
I suppose this injection is not necessary anymore. We could remove this and the relevant import.